### PR TITLE
docs: add ‘getting started’ instructions to README

### LINKED ISSUE #41: Add some ‘Running locally’ instructions to the README

### PR DESCRIPTION
**Pull request by andylolz**
Fixes #41; refs #34.

This provides some instructions for getting started locally.

-----------------

## Pull request checklist

- [x] I have linked my PR to an issue
- [x] I’ve used [conventional commits](https://github.com/FullFact/automation-docs/wiki/Conventional-commits)
- [x] My branch is up-to-date with `main`
- [ ] Where appropriate, I have added or updated tests
- [x] Where appropriate, I have [updated documentation](https://github.com/FullFact/automation-docs/) to reflect my changes

### CODE REVIEW

**Comment by dcorney** [comment]
This looks good! Except when I run `dev_start.sh` (after `dev_setup.sh`) I get:
```
Error: While importing 'raphael_backend_flask.app', an ImportError was raised:

Traceback (most recent call last):
  File "/Users/davidcorney/fullfact/health-misinfo-shared/env/lib/python3.10/site-packages/flask/cli.py", line 245, in locate_app
    __import__(module_name)
  File "/Users/davidcorney/fullfact/health-misinfo-shared/src/raphael_backend_flask/app.py", line 20, in <module>
    from db import create_database
ModuleNotFoundError: No module named 'db'
```

(The server runs anyway but clicking "Analyse" in the UI throws:
```
Uncaught (in promise) SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
Firefox can’t establish a connection to the server at ws://localhost:4000/ws.
```
which is presumably because of the earlier error?)

**Comment by andylolz** [comment]
@dcorney that error is because this PR depends on a couple of others. The easiest way to test it is to check out [local-dev-demo branch](https://github.com/FullFact/health-misinfo-shared/tree/local-dev-demo).

**Comment by andylolz** [comment]
Okay, yes… There are definitely some problems with this, but hopefully this PR is a useful starting point for the non-docker local setup.

File: dev_setup.sh
Comment (ff-dh) [inline dev_setup.sh:1]: This file I don't think is necessary. The backend will create the database if it does not exist on the first request that would access it (any request). `tools.db` might've been a mistake.

Comment (andylolz) [reply dev_setup.sh:1]: Ahh – nice! You are right. I will remove this, and will modify the README accordingly

**Review by ff-dh — APPROVED** [review]
Force-pushing doesn't bother me, looks good

**Pull request merged**